xcmailr-load-test-suite/pom.xml

```diff
@@ -16,12 +16,21 @@
     <description>XCMailr load test suite using XLT</description>
 
     <properties>
+        <java.version>21</java.version>
```

Comment:
Please revert this change as upgrading to Java21 is a separate issue and not related to adding support for secondary SMTP.

Reply:
Done.